src/components/CreateGameDialog.vue

```diff
@@ -12,7 +12,13 @@
       <v-card-text>
         <div class="d-flex align-center">
           <v-switch v-model="isRanked" label="Ranked" data-cy="create-game-ranked-switch" />
-          <stats-scoring-dialog :show-button-text="false" data-cy="ranked-info-button" />
+          <stats-scoring-dialog :show-button-text="false" data-cy="ranked-info-button">
```

Comment:
Should we remove the outer layer data-cy selector now that it's passed via slot?
```suggestion
          <stats-scoring-dialog :show-button-text="false">
```

Reply:
Yep, missed that. I was in POC mode 😬